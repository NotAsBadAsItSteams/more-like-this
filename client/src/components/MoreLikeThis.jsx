import React from 'react';

const MoreLikeThis = () => (
  <div>
    MoreLikeThis
  </div>
);

export default MoreLikeThis;
