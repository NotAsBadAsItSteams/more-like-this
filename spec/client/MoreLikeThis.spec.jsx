import React from 'react';
import renderer from 'react-test-renderer';
import MoreLikeThis from '../../client/src/components/MoreLikeThis';

describe('MoreLikeThis component', () => {
  test('should match snapshot', () => {
    const component = renderer.create(<MoreLikeThis />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
