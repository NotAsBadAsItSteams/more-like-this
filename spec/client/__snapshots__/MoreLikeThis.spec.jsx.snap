// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreLikeThis component should match snapshot 1`] = `
<div>
  MoreLikeThis
</div>
`;
